internal/database/repository/application.go

```diff
@@ -327,11 +333,11 @@ func (s *PostgresRepository) GetAllApplications(ctx context.Context, filters App
 }
 
 //nolint:all
-func (s *PostgresRepository) GetAppsPerPartitionPerQueue(ctx context.Context, partition, queue string, filters ApplicationFilters) ([]*model.Application, error) {
+func (s *PostgresRepository) GetAppsPerPartitionPerQueue(ctx context.Context, partitionID, queueID string, filters ApplicationFilters) ([]*model.Application, error) {
 	queryBuilder := sql.NewBuilder().
 		SelectAll("applications", "").
-		Conditionp("queue_name", "=", queue).
-		Conditionp("partition", "=", partition).
+		Conditionp("queue_id", "=", queueID).
+		Conditionp("partition_id", "=", partitionID).
```

Comment:
Just a ques, If we filter based on `queue_id` do we need `partition_id` ?

Reply:
Good question
My opinion is, since we have a route being /partition/partition_id/queue/queue_id, it doesn't hurt
We can change the route and remove the partition ID, but I wouldn't want to have route with partition_id and never use that parameter

Reply:
Hmm..🤔 I was wondering if it is worth mirroring the `yunikorn` route parameters then, if we are going to use IDs.